crates/blockifier/src/transaction/execution_flavors_test.rs

```diff
@@ -210,13 +211,13 @@ fn test_simulate_validate_charge_fee_pre_validate(
         validate,
         &fee_type,
     );
-    let result = account_invoke_tx(invoke_tx_args! {
+    let account_tx = account_invoke_tx(invoke_tx_args! {
         max_fee: Fee(10),
         resource_bounds: l1_resource_bounds(10, 10),
         nonce: nonce_manager.next(account_address),
         ..pre_validation_base_args.clone()
-    })
-    .execute(&mut state, &block_context, charge_fee, validate);
+    });
+    let result = account_tx.execute(&mut state, &block_context, charge_fee, validate);
```

Comment:
Why is this split into two lines now rather than directly calling `.execute`? Seems like an unnecessary change to me. The same goes for many places below in this same file.